tests/demo/demo.cpp

```diff
@@ -172,6 +184,11 @@ int WispEntry()
 
 	window->SetResizeCallback([&](std::uint32_t width, std::uint32_t height)
 	{
+		if (curRes.width == width && curRes.height == height)
```

Comment:
should probably be done by the render system.

Reply:
The reason why this is being done there is because first the render system is being resized and afterwards the framegraphs which are two separate calls from demo. Having it here prevents duplicate code.